modules/oadp-api-details.adoc

```diff
@@ -0,0 +1,237 @@
+
+|`podAnnotations`
+|map [ link:https://pkg.go.dev/builtin#string[string] ] link:https://pkg.go.dev/builtin#string[string]
+|Used to add annotations to pods deployed by Operators.
+
+|`podDnsPolicy`
+|link:https://pkg.go.dev/k8s.io/api/core/v1#DNSPolicy[`corev1.DNSPolicy`]
+|Defines the configuration of the DNS of a pod.
+
+|`podDnsConfig`
+|link:https://pkg.go.dev/k8s.io/api/core/v1#PodDNSConfig[`corev1.PodDNSConfig`]
+|Defines the DNS parameters of a pod in addition to those generated from `DNSPolicy`.
+
+|`backupImages`
+|*link:https://pkg.go.dev/builtin#bool[bool]
+|Used to specify whether or not you want to deploy a registry for enabling backup and restore of images.
+
+|`configuration`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#ApplicationConfig[`ApplicationConfig`]
+|Used to define the data protection application's server configuration.
+
+|`features`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#Features[`Features`]
+|Defines the configuration for the DPA to enable the Technology Preview features.
+|===
+
+link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#DataProtectionApplicationSpec[Complete schema definitions for the OADP API].
+
+.BackupLocation
+[options="header"]
+|===
+|Property|Type|Description
+
+|`velero`
+|*link:https://velero.io/docs/v1.9/api-types/backupstoragelocation/[velero.BackupStorageLocationSpec]
+|Location to store volume snapshots, as described in link:https://github.com/openshift/oadp-operator/blob/076f1495f6de65b0e96392cb6ef7cff66cf7defb/docs/config/bsl_and_vsl.md[Backup Storage Locations and Volume Snapshot Locations Customization].
+
+|`bucket`
+| [* link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#CloudStorageLocation[CloudStorageLocation] ]
+| [Technology Preview] Automates creation of a bucket at some cloud storage providers for use as a backup storage location.
+
+[IMPORTANT] +
+==== +
+The `bucket` parameter is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview].
+====
+|===
+
+link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#BackupLocation[Complete schema definitions for the type `BackupLocation`].
+
+.SnapshotLocation
+[options="header"]
+|===
+|Property|Type|Description
+
+|`velero`
+|*link:https://velero.io/docs/v1.9/api-types/volumesnapshotlocation/[velero.VolumeSnapshotLocationSpec]
```

Comment:
could you please change that to point to https://pkg.go.dev/github.com/vmware-tanzu/velero/pkg/apis/velero/v1#VolumeSnapshotLocationSpec so it would be consistent with the rest of the schema link references?